Sources/Apodini/Semantic Model Builder/Model/TypeReflection/EnrichedInfo.swift

```diff
@@ -49,7 +49,8 @@ extension EnrichedInfo {
                         guard !errorDescription.contains(keyword) else {
                             return nil
                         }
-                        preconditionFailure(errorDescription)
+                        return nil
+//                        preconditionFailure(errorDescription)
```

Comment:
Without removing this line, I would get a runtime error shortly after startup. 
@nityanandaz Do you know by any chance which that's the case or is it fine to leave it like this. I did some research but couldn't find the cause

Reply:
I address this issue here https://github.com/Apodini/Apodini/pull/131, https://github.com/Apodini/Apodini/pull/131#pullrequestreview-571524716.

The problem is that `Parameter.defaultValue` is a function type since recently.

Reply:
Thanks! So I'll leave it like that for now